client/src/components/StreamField/scss/_variables.scss

```diff
@@ -31,4 +31,4 @@ $screen-xs-max: 799px !default;
 $screen-sm-min: 800px !default;
 $screen-l-min: 1075px !default;
 $add-panel-gutter: 8px !default;
-$font-sans: 'Open Sans', sans-serif !default;
+$font-sans: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji" !default;
```

Comment:
This will be revisited in a subsequent PR – I just did the bare minimum so we could proceed with the removal of the existing font files.